### ISSUE TITLE
How to change dark / light mode?

### ISSUE THREAD

**Issue by MGforest** [CloudCompare/CloudCompare#1994]
Hi, 

I just downloaded the newest version of CloudCompare 2.13.1 via flatpak on Ubuntu 20.04. 

My computer setup is in dark mode. However, now the newest version of CloudCompare adapted to the global setting and the now the interface is  in dark mode. In general, I like that you made a dark mode. But I can't see many of the function buttons because the are black or dark. 

How can I change back to light mode? I can't find setting or suggestions online.

**Comment by rjanvier** [comment]
Hi, it's an interesting question. Maybe you could look at this [thread](https://www.reddit.com/r/flatpak/comments/y9jmqj/the_general_flatpak_qt_and_gtk_theming_guide/). The QT section should be of particular interest. I haven't tried any of these "tricks" myself, please tell us if it is helpful.

**Comment by MGforest** [comment]
Hi, thank you for the quick reply. The given link gave me some ideas. 
I have managed to adjust to light mode, but only temporarily and via terminal. 
This is what I did:

$ flatpak list
shows all packages, I had only Yaru-dark Gtk Theme  org.gtk.Gtk3theme.Yaru-dark          

$ flatpak install flathub org.gtk.Gtk3theme.Yaru-light
installed light theme

$ sudo flatpak override --filesystem=$HOME/.themes
Give Flatpak apps access to GTK themes location
OR
more precise would be
$ sudo flatpak override org.cloudcompare.CloudCompare --filesystem=$HOME/.themes

then override with 
$ QT_STYLE_OVERRIDE=light flatpak run org.cloudcompare.CloudCompare
OR
$ export QT_STYLE_OVERRIDE=light
$ flatpak run org.cloudcompare.CloudCompare

**Comment by MGforest** [comment]
I also just noticed, with my solution above, the software malfunctions. For example, choosing an active point cloud to work on it, is not noticed by the programm, and plugins don't work.

**Comment by rjanvier** [comment]
Hi. I tried to replicate this on my side, unfortunately I have no change in my theme nor (more fortunately) plugins loading error.

I think you run some other bunch of commands because, for example, I'm pretty sure flatpak installed themes does not go into `$HOME/.themes`. Maybe you can try this [resource]( https://docs.flatpak.org/en/latest/desktop-integration.html) 
But my general advice is that if you do not want a dark theme in one application, maybe you'd better switch to a system wide light theme. If you think something is going wrong with CC icons and dark theme, please send us capture of these problematic icons, thanks.

**Comment by Loris-L** [comment]
Is it possible to activate the dark mode on Windows as well? @dgirardeau

**Comment by dgirardeau** [comment]
Sadly the solution designed for the Linux version does not work well for Windows...

**Comment by rjanvier** [comment]
@Loris-L, the choice of dark/light mode is system dependent I guess (but maybe there is an option in QT)
Flatpak (and snap) package cames with their own quirks because they are isolated from the system and theming is handled somehow by their runtime so @MGforest asked if it's not possible to take advantage of that and apply a light theme to CC even if their system theme is dark.

But maybe the "real issue" here is that @MGforest suggest some icons are not properly handled in the dark them.

**Comment by Loris-L** [comment]
Sorry for my intrusion, my question has nothing to do with this problem, but this issue made me ask if in Windows there is an option to switch dark/light mode on CC, as i'm in the dark theme team 😅. Sadly, at the moment i can't use Linux OS' on my PC, so I have to use CC on Windows

**Comment by MGforest** [comment]
> But maybe the "real issue" here is that @MGforest suggest some icons are not properly handled in the dark theme.

Yes, thank you @rjanvier , that is actually the issue that I have. I would like to use the dark mode in CloudCompare but the visibility is not good. 

> I think you run some other bunch of commands because, for example, I'm pretty sure flatpak installed themes does not go into $HOME/.themes. Maybe you can try this [resource](https://docs.flatpak.org/en/latest/desktop-integration.html)

It could be that this line did nothing. I tried some ideas from the internet and posted what I did here. I apologize for the confusion and lack of profound research.
